.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,19 @@
+blank_issues_enabled: false
+
+contact_links:
+
```

Comment:
I'd add one more link to the top for the Scanner. This way, when someone clicks on the create button the `scanner` tag will be automatically added to the new issue.

```
- name: Report problems or suggest new features for the SonarScanner for .NET
  url: https://community.sonarsource.com/tags/c/sq/10/scanner
  about: Community Forum - SonarScanner for .NET
```

Reply:
That's a good one! While having this, I've removed SL, SC and SQ.

SL doesn't make sense in this repo actually. What do you think about SC and SQ? Should I put them back, or is it enough to link to the Scanner-specific category?

Reply:
Should this one also have the `dotnet` tag?

Reply:
1. I think it's fine to keep just the Scanner and the analyzer issues.

2. Yes, that would be even better. I tried adding both tags yesterday, but I could only do one of them:
- Add a category, subcategory and a single tag. Not all tags will be automatically added to the new topic (either `scanner` or `dotnet` will be missing).
- Add multiple tags (intersection) without a category (e.g. https://discuss.sonarsource.com/tags/intersection/false-positive/dotnet). But then you can't specify the category and the user has to select when he creates the topic. I think that's the less attractive solution, selecting the category and subcategory is more confusing.

I couldn't make both of them work at the same time. AFAIK there's a feature request for it (as well as for creating tag groups), but it's not yet implemented for Discourse.